packages/flutter_tools/lib/src/flutter_manifest.dart

```diff
@@ -13,6 +13,9 @@ import 'base/user_messages.dart';
 import 'base/utils.dart';
 import 'plugins.dart';
 
+/// Whether or not Impeller Scene 3D model import is enabled.
+const bool kIs3dSceneSupported = false;
```

Comment:
@jonahwilliams I hardcoded the flag here and made it so stuff doesn't get added to the models list -- LMK if this is good or if I should maybe wire it up somewhere else.

I see this this kind of thing is done elsewhere, like for `kHotReloadDefault` and `kIconTreeShakerEnabledDefault`.

Reply:
SGTM